src/docs/message-handling.dox

```diff
@@ -0,0 +1,105 @@
+/*! @page message_handling Message handling
+
+@section message_handling_event_handler Message handling using the WT_EVENT_HANDLER
+
+Specific error and other message handling can be configured by passing an
+implementation of WT_EVENT_HANDLER to ::wiredtiger_open or
+WT_CONNECTION::open_session.
+
+For example, both informational and error messages might be passed to an
+application-specific logging function that added a timestamp and logged
+the message to a file, and error messages might additionally be output to
+the \c stderr file stream.
```

Comment:
Sorry a little bit confused on what the example is conveying here. Maybe I am missing on how this might be working. What do we mean when we say `application-specific logging function that added a timestamp and logged the message to a file`, is the timestamp refering to WiredTiger timestamps?

Reply:
I did not work on that specific section as this was existing content but I will try to answer!
What we are saying here is that we can interface WiredTiger with a custom logging system through the `WT_EVENT_HANDLER` interface. You can define your own `handle_error`, `handle_message`, `handle_progress` and `handle_close` functions. You can see an example here: [ex_event_handler.c](https://github.com/wiredtiger/wiredtiger/blob/develop/examples/c/ex_event_handler.c).
By doing so, you have the freedom of customizing messages from WiredTiger, i.e adding a timestamp to each message.